cloud/aws/Makefile

```diff
@@ -64,12 +63,11 @@ run-lambda: ## Run ad-hoc VAST clients from AWS Lambda
 		--function-name $(shell terraform output vast_lambda_name) \
 		--cli-binary-format raw-in-base64-out \
 		--payload '{"cmd":"$(CMD)", "host":"$(shell $(MAKE) get-task-ip):42000"}' \
-		lambda-result.json > /dev/null && \
-	cat lambda-result.json
+		/dev/stdout | jq -s -r .[0].result
```

Comment:
I didn't know you could do that!

Reply:
Gotta love the UNIX everything-is-a-file philosophy. 🤓